crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -41,7 +39,8 @@
 # The remaining 187 examples form our training set.
 # This data set is very small, and we're primarily using it for demonstration purposes.
 # In particular, we'd expect to have access to many more unlabeled tweets in order to
-# train a high performance text model.
+# train a high performance text model. Since the dataset is already small, we skip
```

Comment:
Nit: It's nice to start each sentence on a new line in the markdown since it shows u the same, but it makes for clearer diffs (like how this addition has nothing to do with the previous sentence).